src/events/message.ts

```diff
@@ -21,28 +21,31 @@ export default async (client: AldebaranClient, message: Message) => {
 		: "";
 
 	const author = await client.customUsers.fetch(message.author.id);
+	const interactionUser = message.interaction !== null ? 
+		await client.customUsers.fetch(message.interaction.user.id) : 
+		undefined;
```

Comment:
Considering current Aldebaran's code style, the operators should be placed after the line break.
```suggestion
	const interactionUser = message.interaction !== null
		? await client.customUsers.fetch(message.interaction.user.id)
	    : undefined;
```